### ISSUE TITLE
Feature Calculator: add documentation in config dialog

### ISSUE THREAD

**Issue by imagejan** [knime-ip/knip#449]
It can be difficult to find how certain features are calculated if you're not familiar with the source code and the project organisation (knip, imagej-ops, etc.).

It would be great to have some documentation for the feature sets immediately available in the configuration dialog of the *Feature Calculator* node, similar to the way it is done for the aggregation methods in the *GroupBy* node, e.g. by adding a **Description** tab:

![image](https://cloud.githubusercontent.com/assets/2033938/22655686/30a7475c-ec91-11e6-8bd3-d21c7cf72979.png)

**Comment by tibuch** [comment]
@imagejan I agree. I will redesign the UI of the Feature Calculator and make sure that such a description is available. Thank you for pointing it out :)
